third_party/closure/goog/caja/string/html/htmlparser.js

```diff
@@ -474,11 +474,14 @@ goog.string.html.HtmlParser.prototype.parse = function(handler, htmlText) {
 /**
  * Decodes an HTML entity.
  *
+ * @param {string} fullEntity The full entity including the leading
+ * '&', the trailing ';', and the content in between.
  * @param {string} name The content between the '&' and the ';'.
  * @return {string} A single unicode code-point as a string.
  * @private
  */
-goog.string.html.HtmlParser.prototype.lookupEntity_ = function(name) {
+goog.string.html.HtmlParser.prototype.lookupEntity_ = function(fullEntity,
```

Comment:
It's a bit odd to have a documented method with a required param that does nothing. I think I'd prefer to leave this with just `name`, and just have the replace in  `unescapeEntities_` call `lookupEntity_` correctly with the right thing.